masterlist.yaml

```diff
@@ -1017,6 +1017,15 @@ plugins:
         condition: 'active("DNWeathers.esp")'
 
 ###### Modding Tools - Resources ######
+  - name: 'Sortomatic.esm'
+    url:
+      - link: 'https://www.nexusmods.com/newvegas/mods/41142/'
+        name: 'Sortomatic - Modders Resource'
+    # 1.3
+    clean:
```

Comment:
Please write `# 1.3` below the `clean` tag, so that it's directly above the corresponding cleaning data it belongs to.